src/pages/questionPage.js

```diff
@@ -27,14 +27,24 @@ export const initQuestionPage = () => {
       showCorrectAnswer(answerElement, key, currentQuestion);
     });
   }
+  const nextQuestionCheck = () => {
```

Comment:
Can you try to move this function to a separate util file?

Reply:
Done!